Lombiq.Tests.UI.Samples/.htmlvalidate.json

```diff
@@ -15,5 +15,16 @@
     "wcag/h37": "off",
     "wcag/h67": "off",
     "wcag/h71": "off"
-  }
+  },
+
+  "elements": [
+    "html5",
+    {
+      "form": {
+        "attributes": {
+          "method": [ "get", "post", "GET", "POST" ]
+        }
+      }
+    }
+  ]
```

Comment:
Do we actually need this config and Lombiq.Tests.UI/.htmlvalidate.json or did you add it for testing?

Reply:
It will be needed for Jocó's issue so I see no reason to remove it, especially if his ends up merged sooner which is totally possible. If I were to delete these lines then git will get confused later during merge as one branch adds and another removes the same code. No thanks.

Reply:
OK then.